oeps/oep-0044-bp-changelogs.rst

```diff
@@ -0,0 +1,294 @@
+
+To make it easier for users and contributors to see precisely what notable changes have been made between each release (or version) of the project.
+
+Who needs a changelog?
+~~~~~~~~~~~~~~~~~~~~~~
+
+People do. Whether consumers or developers, the end users of software are human beings who care about what's in the software. When the software changes, people want to know why and how.
+
+Types of Repositories
+---------------------
+
+For the purposes of this OEP, we will refer to two types of repositories.
+
+**Library Repository**: A library is a repository containing code that is versioned. These libraries often produce packages that are published to PyPI (for Python) or npm (for JavaScript).
+
+**Application or Non-Library Repository**: Any repository that does not contain versioned code will be referred to as an application or non-library respository.
+
+Decisions
+=========
+
+All Open edX repositories should contain a CHANGELOG.rst. There are a variety of acceptable formats, depending on the type of repository and other factors, as discussed below.
+
+Changelogs and Changelog Entries
+--------------------------------
+
+Here are some recommendations for formatting changelogs and changelog entries, which have been taken from `keepachangelog.com`_.
+
+Guiding Principles
+~~~~~~~~~~~~~~~~~~
+
+* Changelogs are for humans, not machines.
+* There should be an entry for every single version [for libraries].
+* The same types of changes should be grouped.
+* Versions and sections should be linkable.
+* The latest version comes first.
+* The release date of each version is displayed.
+* Mention whether you follow Semantic Versioning.
+
+Types of Changes
+~~~~~~~~~~~~~~~~
+
+* **Added** for new features.
+* **Changed** for changes in existing functionality.
+* **Deprecated** for soon-to-be removed features.
+* **Removed** for now removed features.
+* **Fixed** for any bug fixes.
+
+These make great section headers, but similar language could also be used in messages to clarify the type of change.  For commits, you may want to use "adds" in place of "added", etc.
+
+The recommendation also includes "**Security** in case of vulnerabilities", but that is being called out separately because the Security Group needs to manage the timing of adding any such designation to any public changelog.
+
+Other Recommendations
+~~~~~~~~~~~~~~~~~~~~~
+
+* Format dates as YYYY-MM-DD. As an example, 2020-03-21.
+* If you do nothing else, list deprecations, removals, and any breaking changes in your changelog.
+* It is both acceptable and appropriate to clean up changelog entries at any time to add clarifications or customize messaging for the audience of the changelog when needed.
+
+Example Changelog
+~~~~~~~~~~~~~~~~~
+
+Here is an example CHANGELOG.rst.
+
+.. note:: Although this example includes versions, based on the decisions below our changelog entries would typically only include dates, because our versioned changelogs are managed using GitHub Releases.
```

Comment:
I thought the recommendations here were to either have explicit text in changelog.rst, or a link to the GitHub releases page?  But this sentence seems to anticipate a changelog file that has dates, but no version numbers?

Reply:
I removed the versions from the example and updated the intro line, so hopefully this is more clear.